### PR TITLE
Coding question: JS reference

### PR DESCRIPTION
**Pull request by MihaylovNikitos**
Added the coding question about reference in JavaScript.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,6 +191,22 @@ foo.push(1);
...
+  obj1.item = "changed";
+  obj2 = {item: "changed"};
+}
+
+var num = 10;
+var obj1 = {item: "unchanged"};
+var obj2 = {item: "unchanged"};
+
+change(num, obj1, obj2);
+```
```
Comment (arthurvr) [inline README.md]: Can you try to make this code snippet consistent with the rest of the code examples? Spacing, single quotes, `{` shouldn't be on it's own line, ...

**Comment by MihaylovNikitos** [comment]
Sorry for '{'. I did code correctly.

**Comment by darcyclarke** [comment]
@MihaylovNikitos can you rebase? We've made some merges and they've unfortunately made this out-of-date.

**Comment by richgilbank** [comment]
@MihaylovNikitos Also, can you clarify _where_ you're asking what those values are? i.e. At the end of (inside) the `change` function, vs the end of the snippet?

**Comment by arthurvr** [comment]
ping @MihaylovNikitos

**Comment by richgilbank** [comment]
@MihaylovNikitos I'm going to close this for now due to the inactivity - feel free to reopen when you get a chance to rebase and answer the above question.

**Pull request closed without merging**